# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#result definition
std_msgs/Header header
actionlib_msgs/GoalStatus status
arm_navigation_msgs/MoveArmResult result
